service/src/main/java/greencity/filters/RatingStatisticsSpecification.java

```diff
@@ -63,7 +64,7 @@ private Predicate getEventNamePredicate(Root<RatingStatistics> root, CriteriaBui
         SearchCriteria searchCriteria) {
         List<RatingCalculationEnum> enumValues = Arrays.asList(RatingCalculationEnum.values());
         List<RatingCalculationEnum> selectedEnums = enumValues.stream()
-            .filter(x -> x.toString().toLowerCase().contains(((String) searchCriteria.getValue()).toLowerCase()))
+            .filter(x -> x.toString().equalsIgnoreCase(((String) searchCriteria.getValue())))
```

Comment:
We will have max 1 element in the `selectedEnums`  list
We compare every `RatingCalculationEnum` element with the same string `searchCriteria.getValue()` and equalsIgnoreCase could be true for max 1 element

Reply:
fixed